packages/renderer/src/lib/dashboard/ProviderInitUtils.ts

```diff
@@ -0,0 +1,26 @@
+/**********************************************************************
+ * Copyright (C) 2023 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+import GearIcon from '../images/WrenchIcon.svelte';
+import StartIcon from '../images/StartIcon.svelte';
+
+export const InitializeOnlyMode = 'Initialize';
+export const InitializeAndStartMode = 'Initialize and Start';
```

Comment:
```suggestion
export const InitializeAndStartMode = 'Initialize and start';
```

SUPER SMALL NITPICK I don't even know why I'm commenting...

It should be `Initialize and start`

Since when it shows it's "Initialize and Start Podman" which is incorrect grammar. Start shouldn't be capitalized.

Reply:
@lstocchi if you can update the title so we can go ahead and merge the PR ?